doc/pages/user/015-astarte_dashboard.md

```diff
@@ -0,0 +1,394 @@
+  device's connections to Astarte, the dates of first registration and credentials request, and the
+  dates of last connection and disconnection, if available.
+- **Live Events**: a section that reports live events regarding the device. It makes use of
+  [Astarte Channels](052-using_channels.html) and displays the device's connections, exchanged data,
+  and errors, as they happen in real-time.
+
+## Groups
+
+[Groups](065-managing-groups.html) are logical collections of devices to ease the management and
+querying of devices.
+
+You can navigate to this section thanks to the Dashboard's side menu. A list of all existing groups
+is displayed, together with the number of total and connected devices for each group.
+
+### Creating a group
+
+From the Group list, clicking on the **Create a new group** button will load a dedicated page to
+setup the new group.
+
+You are required to specify a name for the group and to select at least one device that will belong
+to it. Indeed, note that a group must contain at least one device to exist.
+
+To confirm the creation of the new group, click the **Create group** button.
+
+### Editing groups
+
+From the Group list, you can select a group to view its configuration on a dedicated page. Here you
+can review the list of devices that belong to it.
+
+To remove a device from a group, click on the Delete icon next to it.
+
+To add a device to a group, you can first navigate to the device's page and then add it to a group
+from there.
+
+To delete a group, remove all devices that belong to it and the group will automatically cease to
+exist.
+
+## Blocks
+
+[Blocks](https://docs.astarte-platform.org/flow/snapshot/0003-blocks.html) are computation units
+that can be chained together to define a logical computation topology.
+
+You can navigate to this section thanks to the Dashboard's side menu. Here is a list of all existing
+blocks, both custom and native ones: the former ones are those defined by you, the latter ones are
+those provided by Astarte and are displayed with a `native` label.
+
+### Creating blocks
+
+From the Block list, clicking on the **Create** button will load a dedicated page to define a new
+block.
+
+Here you can define the block by specifying:
+
+- A `name` for the block.
+- A `type`, between `Producer`, `Consumer` or `Producer & Consumer`, depending on how the block
+  should treat messages and connect to other blocks.
+- A `schema`, reporting the JSON Schema definition of how a configuration should look like for the
+  block.
+- A `source`, containing the block's implementation, written in the
+  [DSL format](https://docs.astarte-platform.org/flow/snapshot/0010-defining-a-pipeline.html) for
+  Pipelines.
+
+Confirm the block's creation by clicking on the **Create new block** button.
+
+To learn more about blocks definition you can read the
+[documentation's reference](https://docs.astarte-platform.org/flow/snapshot/0003-blocks.html).
+
+### Editing blocks
```

Comment:
As for now blocks cannot be edited

Reply:
You're right, I went with `managing blocks`